.github/workflows/check-examples.yml

```diff
@@ -42,12 +42,13 @@ jobs:
         python-version: 3.9
     - name: install python requirements for notebooks
       run: |
+        cd pytket-examples-book
         python -m pip install --upgrade pip
         python -m pip install pre-commit==3.0.0
         python -m pip install wheel
-        python -m pip install -r requirements.txt
-        python -m pip install -c constraints.txt jupyter openfermion plotly
```

Comment:
I have removed pytest and openfermion here and added them to the `example_requirements.txt` file.

These are needed for the examples `creating_backends` and ucc_vqe`.